### PR TITLE
Fix issue #28 fedora 27 build

### PR DESCRIPTION
**Pull request by zbeekman**
This should resolve everything from SOW1 except the new profiles.

### CODE REVIEW

**Comment by zbeekman** [comment]
Specifically, I believe we can close/sign off on #32 after review from @robnagler and @dtabell. I hope to complete #4 this afternoon/evening and have it ready for review tomorrow.

**Comment by zbeekman** [comment]
(Also, once we have more tests & dynamic memory allocation from SOW2, the CMake/CTest code will end up being simplified and the supporting script for running the test can be removed. There are some additional CMake best practices that I would like to implement/cleanup during SOW2 as well.)

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+-------------------
+
+1. Download and install [vagrant](https://www.vagrantup.com).
+
+2. Download and install [VirtalBox](https://www.virtualbox.org).
+
+3. Obtain and launch the RadiaSoft development environment:
+
+```bash
+curl radia.run | bash -s vagrant-sirepo-dev
```
Comment (robnagler) [inline INSTALL.md]: The order isn't right. Perhaps just link to https://github.com/radiasoft/sirepo/wiki/Development#mac-install

File: tests/zgoubi-integration-test.sh.template [hunk 2]
```diff
@@ -35,7 +33,7 @@ fi
 
 footer=10
 res_file_length=$(wc -l zgoubi.res.expected | awk '{print $1;}')
```
Comment (robnagler) [inline tests/zgoubi-integration-test.sh.template]: just fyi: (( header=$(wc -l < zgoubi.res.expected) - footer ))

Thanks: I didn't know that you didn't need "$" inside (( )).

**Review by robnagler — COMMENTED** [review]
I can accept as is, or you can modify and re-push.

**Comment by zbeekman** [comment]
@robnagler I implemented both of your suggestions, plus some other improvements (IMO) in INSTALL.md. Let me know if these look OK to you, I won't be offended if you want to revert one or both of the commits I just pushed.

Otherwise, this is good to go as far as I am concerned.

**Pull request merged**